cogs/utility.py

```diff
@@ -12,246 +12,265 @@
+									   f"/master/all-files/{filename})")
+				else:
+					rdir = str(all_files[accessory].get("rdir", None))
+					em.add_field(name=name,
+								 value=f"Author: {author}\n[Github Link](https://github.com/{repo}/tree/master/all-"
+									   f"files/{rdir})")
+		await ctx.send(embed=em)
+
+	@commands.command()
+	async def datetime(self, ctx, tz=None):
+		"""Get the current date and time for a time zone or UTC."""
+
+		# Send the current datetime
+		now = datetime.datetime.now(tz=pytz.UTC)
+		if tz:
+			try:
+				now = now.astimezone(pytz.timezone(tz))
+			except:
+				em = discord.Embed(color=utils.random_color())
+				em.title = "Invalid timezone"
+				em.description = f'Please search for the list of timezones on google.'
+				return await ctx.send(embed=em)
+		await ctx.send(f'It is currently {now:%A, %B %d, %Y} at {now:%I:%M:%S %p}.')
+
+	@commands.command(aliases=['wikipedia'])
+	async def wiki(self, ctx, *, query):
+		"""Search up something on wikipedia"""
+		em = discord.Embed(title=str(query), color=utils.random_color())
+		em.set_footer(text='Powered by wikipedia.org')
+		try:
+			# Search the query on wikipedia
+			result = wikipedia.summary(query)
+			if len(result) > 2000:
+				em.description = f"Result is too long. View the website [here](https://wikipedia.org/wiki/" \
+								 f"{query.replace(' ', '_')}), or just google the subject."
+				return await ctx.send(embed=em)
+			em.description = result
+			await ctx.send(embed=em)
+		except wikipedia.exceptions.DisambiguationError as e:
+			options = '\n'.join(e.options)
+			em.description = f"**Options:**\n\n{options}"
+			await ctx.send(embed=em)
+		except wikipedia.exceptions.PageError:
+			em.description = 'Error: Page not found.'
+			await ctx.send(embed=em)
+
+	@commands.command()
+	@commands.cooldown(1, 15, BucketType.user)
+	async def feedback(self, ctx, *, idea: str):
+		"""Suggest an idea, or complain an issue."""
+		suggest = self.bot.get_channel(612530117830246400)
+		em = discord.Embed(color=utils.random_color())
+		em.title = f"{ctx.author} | User ID: {ctx.author.id}"
+		em.description = idea
+		try:
+			i = ctx.guild.icon_url or self.bot.user.avatar_url
+			em.set_footer(text=f"From {ctx.author.guild} | Server ID: {ctx.author.guild.id}",
+						  icon_url=i)
+		except:
+			em.set_footer(text=f"Received from a Private channel.")
+		await suggest.send(embed=em)
+		await ctx.send("Your idea has been successfully sent to support server. Thank you!")
+
+	@commands.command(aliases=["server", "server_list", "server-list"])
+	@commands.cooldown(1, 15, BucketType.user)
+	async def _search(self, ctx, *, region: str):
+		"""To get a list of servers according to region."""
+		#Made By: AwesomeLogic
+		url = f"https://awesomelogic.herokuapp.com/api/{region}"
+		async with aiohttp.ClientSession() as session:
+			async with session.get(url) as resp:
+				data = json.loads(eval(await resp.read()))
```

Comment:
This can give error if the region given by user is not valid.
You should add try except around the aiohttp session and call to catch any exception and then send a message to the in the channel to go to https://awesomelogic.herokuapp.com/api/regions